compiler/rustc_parse/src/parser/ty.rs

```diff
@@ -694,9 +694,14 @@ impl<'a> Parser<'a> {
         &mut self,
     ) -> PResult<'a, (ThinVec<PreciseCapturingArg>, Span)> {
         let lo = self.token.span;
-        let (args, _) = self.parse_unspanned_seq(
-            &TokenKind::Lt,
```

Comment:
I can't figure out how `parse_seq_to_before_tokens` eats the leading `<` (`parse_unspanned_seq` checked for leading `<` after all).

Reply:
there's an `expect_lt()` call right above it on line 697 😄